src/app/address/address.component.html

```diff
@@ -0,0 +1,108 @@
+<div class="container">
```

Comment:
The HTML formatting below, where the `>` and the `/>` are on a separate line looks a bit strange to me. Is this intentional or something that's being don by a plugin in your IDE?

Reply:
Yes this is being done by a plugin called "Prettier".